src/main/handlers/recentProjectsHandler.ts

```diff
@@ -3,20 +3,28 @@ import fs from 'fs/promises';
 import { Project, RecentProject } from '../../sharedTypes';
 import { appDataStoragePath } from '../util';
 import retrieveMetadata from './projectMetadataHandler';
+import makeRecentProject from '../../sharedUtils';
 
 const RECENT_PROJECTS_PATH = path.join(
   appDataStoragePath(),
   'recentProjects.json'
 );
 
 export const writeRecentProjects: (
-  recentProjects: Project[]
+  recentProjects: RecentProject[]
 ) => Promise<void> = async (recentProjects) => {
   const recentProjectsJson = JSON.stringify(recentProjects);
 
   await fs.writeFile(RECENT_PROJECTS_PATH, recentProjectsJson);
 };
 
+/**
+ * Type for project persisted to disk in the recent projects list -
+ * includes the intersection of project and recent project fields (i.e. no metadata
+ * as this can go stale)
+ */
```

Comment:
Maybe increase the clarity of the comment. I'm still not entirely sure of the difference between a RecentProject and a PersistedRecentProject. Or perhaps change the type name?